.github/workflows/build.yml

```diff
@@ -0,0 +1,17 @@
+name: Build
+on: 
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: rancher/dapper:v0.6.0
+    steps:
+      # rancher/dapper:0.6.0 doesn't have a git installation. This is necessary to run actions/checkout
+      - name: Add Git
```

Comment:
Looks like I was able to get `act` to run a local build without adding Git.

Reply:
When testing on a private repo, I got an error. If we don´t install git, the checkout action won't go through all the steps needed to set up the working directory properly. This will result in an error later on running dapper build.

This are the steps it runs without git installation:
![image](https://github.com/rancher/csp-adapter/assets/166652561/8b127ec3-12af-4c2d-8e53-257c7ced1e3c)

This are the steps it runs with git installation:
![image](https://github.com/rancher/csp-adapter/assets/166652561/139c6a73-6d62-4f58-a0fc-cf24208c3ac7)

The error:
![image](https://github.com/rancher/csp-adapter/assets/166652561/d1b507fa-6d16-42ad-a602-95b3458e49f1)